docs/portal/image-pull-secrets/viewing-image-pull-secret-information.md

```diff
@@ -9,7 +9,7 @@ The following procedure describes how to view information about ImagePullSecrets
 
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
-1                                                                                                       | The Basic Information area shows basic information about the ImagePullSecret.                                     |
-| 2                        | Click this button to return to the ImagePullSecrets page.                                                     |
-| 3                        | Click this button to update the ImagePullSecret.                                                           |
-| 4                        | Click this button to delete the ImagePullSecret.                                                           |
\ No newline at end of file
+<p style="text-align: center;">1</p>                                                                                                       | The Basic Information area shows basic information about the ImagePullSecret.                                     |
```

Comment:
The preview shows there are errors. Please check
![image](https://user-images.githubusercontent.com/43203744/105329991-bf132d00-5c0c-11eb-91c2-1366c1422080.png)